presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -0,0 +1,98 @@
+====================
+SQL Server Connector
+====================
+
+The SQL Server connector allows querying and creating tables in an
+external SQL Server database. This can be used to join data between
+different systems like SQL Server and Hive, or between two different
+SQL Server instances.
+
+Configuration
+-------------
+
+To configure the SQL Server connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``sqlserver.properties``, to
+mount the SQL Server connector as the ``sqlserver`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=sqlserver
+    connection-url=jdbc:sqlserver://[serverName[\instanceName][:portNumber]]
+    connection-user=root
+    connection-password=secret
+
+Multiple SQL Server Databases or Servers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The SQL Server connector can only access a single database within
+a SQL Server server. Thus, if you have multiple SQL Server databases,
+or want to connect to multiple SQL Server servers, you must configure
+multiple instances of the SQL Server connector.
+
+To add another catalog, simply add another properties file to ``etc/catalog``
+with a different name (making sure it ends in ``.properties``). For example,
+if you name the property file ``sales.properties``, Presto will create a
+catalog named ``sales`` using the configured connector.
+
+Querying SQL Server
+-------------------
+
+The SQL Server connector provides a schema for every SQL Server schema.
+You can see the available SQL Server schemas by running ``SHOW SCHEMAS``::
+
+    SHOW SCHEMAS FROM sqlserver;
```

Comment:
I think its fine to have it here as an example. In case, you want I can provide a link to `show schemas`